docs/docs/1.7.x/security/api-server-auth.md

```diff
@@ -1,156 +1,167 @@
-$ kubectl delete secret admin-user-token -n kuma-namespace
-```
+kumactl generate user-token --name=john --group=team-a --valid-for=24h | jwt
 
-2. Disable bootstrap of the token
-[Configure a control plane](../documentation/configuration.md) with `KUMA_API_SERVER_AUTHN_TOKENS_BOOTSTRAP_ADMIN_TOKEN` set to `false`.
-:::
-::: tab "Universal"
-1. Delete `admin-user-token` Global Secret
-```sh
-kumactl delete global-secret admin-user-token
-```
+To verify on jwt.io:
 
-2. Disable bootstrap of the token
-[Configure a control plane](../documentation/configuration.md) with `KUMA_API_SERVER_AUTHN_TOKENS_BOOTSTRAP_ADMIN_TOKEN` set to `false`.
-:::
-::::
+https://jwt.io/#id_token=eyJhbGciOiJSUzI1NiIsImtpZCI6IjEiLCJ0eXAiOiJKV1QifQ.eyJOYW1lIjoiam9obiIsIkdyb3VwcyI6WyJ0ZWFtLWEiXSwiZXhwIjoxNjM2ODExNjc0LCJuYmYiOjE2MzY3MjQ5NzQsImlhdCI6MTYzNjcyNTI3NCwianRpIjoiYmYzZDBiMmUtZDg0MC00Y2I2LWJmN2MtYjkwZjU0MzkxNDY4In0.XsaPcQ5wVzRLs4o1FWywf6kw4r2ceyLGxYO8EbyA0fAxU6BPPRsW71ueD8ZlS4JlD4UrVtQQ7LG-z_nIxlDRAYhx4mmHnSjtqWZIsVS13QRrm41zccZ0SKHYxGvWMW4IkGwUbA0UZOJGno8vbpI6jTGfY9bmof5FpJJAj_sf99jCaI1H_n3n5UxtwKVN7dXXD82r6axj700jgQD-2O8gnejzlTjZkBpPF_lGnlBbd39S34VNwT0UlvRJLmCRdfh5EL24dFt0tyzQqDG2gE1RuGvTV9LOT77ZsjfMP9CITICivF6Z7uqvlOYal10jd5gN0A6w6KSI8CCaDLmVgUHvAw
 
-### Token revocation
+✻ Header
+{
+  "alg": "RS256",
+  "kid": "1",
+  "typ": "JWT"
+}
 
-Kuma does not keep the list of issued tokens. Whenever the single token is compromised, we can add it to revocation list so it's no longer valid.
+✻ Payload
+{
+  "Name": "john",
+  "Groups": [
+    "team-a"
+  ],
+  "exp": 1636811674,
+  "nbf": 1636724974,
+  "iat": 1636725274,
+  "jti": "bf3d0b2e-d840-4cb6-bf7c-b90f54391468"
+}
+   Issued At: 1636725274 11/12/2021, 2:54:34 PM
+   Not Before: 1636724974 11/12/2021, 2:49:34 PM
+   Expiration Time: 1636811674 11/13/2021, 2:54:34 PM
 
-Every user token has its own ID which is available in payload under `jti` key. You can extract ID from token using jwt.io or [`jwt-cli`](https://www.npmjs.com/package/jwt-cli) tool. Here is example of `jti`
-```
-0e120ec9-6b42-495d-9758-07b59fe86fb9
+✻ Signature XsaPcQ5wVzRLs4o1FWywf6kw4r2ceyLGxYO8EbyA0fAxU6BPPRsW71ueD8ZlS4JlD4UrVtQQ7LG-z_nIxlDRAYhx4mmHnSjtqWZIsVS13QRrm41zccZ0SKHYxGvWMW4IkGwUbA0UZOJGno8vbpI6jTGfY9bmof5FpJJAj_sf99jCaI1H_n3n5UxtwKVN7dXXD82r6axj700jgQD-2O8gnejzlTjZkBpPF_lGnlBbd39S34VNwT0UlvRJLmCRdfh5EL24dFt0tyzQqDG2gE1RuGvTV9LOT77ZsjfMP9CITICivF6Z7uqvlOYal10jd5gN0A6w6KSI8CCaDLmVgUHvAw
 ```
 
-Specify list of revoked IDs separated by `,` and store it as `GlobalSecret` named `user-token-revocations`
+### Token revocation
+
+Kuma doesn't keep the list of issued tokens. To invalidate the token, you can add it to a revocation list.
+Every user token has its own ID. As you saw in the previous section, it's available in payload under `jti` key.
+To revoke tokens, specify list of revoked IDs separated by `,` and store it as `GlobalSecret` named `user-token-revocations`
 
 :::: tabs :options="{ useUrlFragment: false }"
 ::: tab "Kubernetes"
```

Comment:
Maybe in these examples you could use `jwt` .e.g: `pbpaste | jwt decode - --json | jq -r '.payload.jti'`